requirements/core.txt

```diff
@@ -8,7 +8,7 @@ cartopy>=0.12
 cf-units>=2
 cftime==1.1.3
 dask[array]>=2  #conda: dask>=2
-matplotlib
+matplotlib<3.3
```

Comment:
@stephenworsley Happy with the temporary pin here, and we can address these test failures seperately